jumanji/environments/logic/rubiks_cube/env.py

```diff
@@ -86,7 +85,7 @@ def __init__(
         self,
         time_limit: int = 200,
         reward_fn: Optional[RewardFn] = None,
-        env_viewer: Optional[RubiksCubeViewer] = None,
+        env_viewer: Optional[Viewer[State]] = None,
```

Comment:
Could we call the environment viewer `viewer` to be consistent with the other environments?
```suggestion
        viewer: Optional[Viewer[State]] = None,
```

Reply:
(I changed it to be consistent)